docs/upgrade/4.0.rst

```diff
@@ -0,0 +1,69 @@
+.. _upgrade-to-3.8:
+
+###################
+4.0.0 release notes
+###################
+
+This release of django CMS is a complete rewrite of the core, hugely simplyfying what django-cms does
+out of the box. The main reasons for the changes:
+    - Limitations with Publishing where only 2 versions can ever exist
+    - Too many "opinions" of how parts of the CMS should work
```

Comment:
```suggestion
    - Too many "opinions" of how parts of the CMS should work baked in
```

Reply:
I'd probably add a comma there to make that "work, baked in"